src/courtroom.cpp

```diff
@@ -1538,7 +1538,11 @@ void Courtroom::append_server_chatmessage(QString p_name, QString p_message,
 
 void Courtroom::on_chat_return_pressed()
 {
-  if (ui_ic_chat_message->text() == "" || is_muted)
+  if (is_muted)
+    return;
+
+  // You can only blankpost w/o any input if the last character who spoke is yours
+  if (ui_ic_chat_message->text() == "" && m_chatmessage[CHAR_ID].toInt() != m_cid)
```

Comment:
So then people will just add a space. Right?

Reply:
Only for their first message. Subsequent messages don't need a space

Reply:
Why do we require a space for the first message?

Reply:
Just because you're asking that now we won't